src/main/java/io/stargate/sgv2/jsonapi/api/model/command/impl/CreateCollectionCommand.java

```diff
@@ -34,7 +34,8 @@ public record CreateCollectionCommand(
   public record Options(
 
       // limit of returned documents
-      @Schema(
+      @JsonInclude(JsonInclude.Include.NON_NULL)
```

Comment:
We could also use `JsonInclude.Include.NON_EMPTY` which would exclude empty `Collection`s.

Reply:
Ah., got it I will change it.